### PR TITLE
Instructions for contributing

### PR DESCRIPTION
**Pull request by pireddag**

### CODE REVIEW

**Comment by mgubi** [comment]
I do not think that the first step is needed (no fork)

File: README.md [hunk 1]
```diff
@@ -22,8 +22,36 @@ To automatically regenerate all the web pages  use `Tools->Web->Create/Update we
...
+    ```
+    Already on 'main'
+    Your branch is up to date with 'origin/main'.
+    ```
+    )
+    * fetch upstream
+    * merge upstream/main
+    * push
+    
+Typically it should just be necessary to modify or add to only the `.tm` sources in the src/directory---and modify accordingly index.tm. The html files are then generated automatically after the pull request is merged. Once the changes are pulled in they will become immediately publicly visible on the website.
```
Comment (mgubi) [inline README.md]: This last sentence is not (yet) true. There is no yet an automatic mechanism which produces the pages.

**Review by mgubi — COMMENTED** [review]
Overall seems ok. Thanks! You might want to have also these instructions in one of the articles on the blog.

**Comment by pireddag** [comment]
Ok I changed it with the note (that I can remove) that there is the plan to automate.

**Comment by pireddag** [comment]
I also saw now that if I make a pull request, what is pulled is the repository at the moment of the approval, not at the moment of the request. So this is the way that review works.

**Comment by pireddag** [comment]
I'll try to refine the instructions (now it is going to quickly in the details, without explanation of what one is doing) and write them in in .tm format for the blog,

**Review by mgubi — APPROVED** [review]
Great work Giovanni. We could also mention that it is possible to browse the .tm files remotely, but I still haven't checked myself (see the message of Joris in the mailing list).

**Comment by pireddag** [comment]
I have added instructions for browsing the .tm files locally in another pull request (remotely I still have to check too). I think it would be good if we discuss on the mailing list the contribution policy. But I write something here too :-) :  the mechanism of the blog (TeXmacs plus pull requests) is such that people have to work more in order to contribute, and I think that it is necessary to reduce as much as possible the amount of work in order to get the maximum possible participation (which I am convinced is higher than the current).

**Pull request merged**